app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt

```diff
@@ -76,7 +76,7 @@ class ContactPatientBottomSheet : BottomSheetActivity(), ContactPatientUi, Conta
 
   private val permissionResults: Subject<ActivityPermissionResult> = PublishSubject.create()
 
-  private val dialogEvents: PublishSubject<ContactPatientEvent> = PublishSubject.create()
+  private val otherEvents: PublishSubject<ContactPatientEvent> = PublishSubject.create()
```

Comment:
Decided to name them `hotEvents` because this is essentially a "hot" observable (http://reactivex.io/documentation/observable.html). Accepting better naming suggestions 😓.

Reply:
`hotEvents` is like saying "Today's hot news" 😄.
Let's just keep it as `otherEvents`. I couldn't think of anything else either.